htlcswitch/mailbox.go

```diff
@@ -422,7 +430,7 @@ func (m *memoryMailBox) wireMailCourier() {
 
 // pktMailCourier is a dedicated goroutine whose job is to reliably deliver
 // packet messages.
-func (m *memoryMailBox) pktMailCourier() {
+func (m *memoryMailBox) pktMailCourier(ctx context.Context) {
```

Comment:
This `ctx` is not caught in the `quit` below?

Reply:
indeed - rebase error, sorry about that. Thanks for the catch 🙏